rust/src/adapter/datadog.rs

```diff
@@ -15,7 +15,7 @@ use url::Url;
 
 use crate::{Event, Log, Metric, TraceEvent};
 
-use super::{
+pub use super::{
```

Comment:
```suggestion
use super::datadog_formatter::{DatadogFormatter, Span, Trace};
pub use super::{
 Adapter, AdapterHandle, AdapterMetadata,
};
```

Reply:
(not 100% sure about that particular suggestion -- only did on github so worth checking :laughing:)